tools/src/test/scala/com/nvidia/spark/rapids/tool/qualification/AppFilterSuite.scala

```diff
@@ -339,4 +369,64 @@ class AppFilterSuite extends FunSuite {
       }
     }
   }
+
+  test("standalone 1-oldest-per-app-name") {
+    val expected = Array("ndshours18", "ndsweeks2", "nds86")
+    testAppStartTimeAndPerAppName(appsFullWithFsToTest, "1-oldest-per-app-name", 3, expected)
```

Comment:
I'd prefer to see a list of apps. with more then just one that has 2 of the same app name.  one should have > 2 and another should have at least 2.

Checking expected is nds86 doesn't test oldest or newest. the appid should have unique name: local-16261043003${app.uniqueId} so lets check that to make sure we got the right nds86 app

Reply:
Updated the tests where we compare both appId and appName. So we can be certain the oldest/newest is the right one as expected.